lib/steps.js

```diff
@@ -126,22 +127,18 @@ const steps = {
 
     await steps.commit(preCfg)
   },
-  run: (cmd, successMessage, dryRun, quiet) => {
+  run: async (cmd, successMessage, dryRun, quiet) => {
     const promise = dryRun
       ? Promise.resolve('')
       : execAsync(cmd).then(({ stdout }) => stdout)
 
-    if (successMessage) {
-      promise.then((stdout) => {
-        if (!quiet) {
-          console.log(chalk`${successMessage} {blue > ${cmd}}`)
-        }
+    const stdout = await promise
```

Comment:
Why not just await the above expression instead of assigning it to `promise` now? 🤔

Reply:
sure! guess I didn't thought of that..

Reply:
I thought of doing

```ts
const stdout = await (dryRun
  ? Promise.resolve('')
  : execAsync(cmd).then(({ stdout }) => stdout))
```

but lint complains about the shadowing of `stdout`, what do you think?

Reply:
meh, I'll rename the variable inside the `.then`